web/src/components/MemoEditor/Editor/index.tsx

```diff
@@ -158,13 +158,19 @@ const Editor = forwardRef(function Editor(props: Props, ref: React.ForwardedRef<
 
       const cursorPosition = editorActions.getCursorPosition();
       const prevContent = editorActions.getContent().substring(0, cursorPosition);
+      const lines = prevContent.split("\n");
+      const lastLine = lines[lines.length - 1];
```

Comment:
Non blocker: is there a benefit to getting the count of the `\n` and then using the `editorActions` `getLine` function which does the same as this code but does not keep the array of lines (that doesn't get used after this point) in memory. Thinking of scenarios with many many lines of text in the memo so a large array will be stored.

The benefit will realistically be minute so as mentioned, this is a non blocker and no change needed, but though its worth pondering